src/sql_query.ts

```diff
@@ -100,7 +100,7 @@ export default class SqlQuery {
         this.target.rawQuery = query
             .replace(/\$timeSeries/g, SqlQuery.getTimeSeries(dateTimeType))
             .replace(/\$timeFilter/g, timeFilter)
-            .replace(/\$table/g, this.target.database + '.' + this.target.table)
+            .replace(/\$table/g, this.target.database + '.' + '`' this.target.table + '`')
```

Comment:
missed `+` char betwen quotes and `this.target.table`

Reply:
database could also contain space character

Reply:
@hagen1778 I have made the various changes.

Regarding the space character usage, I admin it is not very nice to have names containing space and as long as clickhouse allows it, It is better to make the plugin working with.
Moreover, we are not going to replace spaces with something different as we prefer to keep the exact name linked to our internal naming to avoid having a corresponding table located somewhere.

Reply:
Just wanted to say, that having space characters in names rather rare situation. And if so we need to apply quotes only when it is needed (there are other reasons in my prev comment). I believe, it would be better to create new method to check whether passed argument must be quoted. That method could be also used for other entities with possible spaces in names, described in CH specification.

Reply:
You mean, only appling quote at runtime by detecting argument ? For example (in pseudo langage): if arg contains space, then add quote else do nothing.

That could be an interesting idea